playgrounds/demo/vite.config.js

```diff
@@ -11,7 +11,7 @@ export default defineConfig({
 		inspect(),
 		svelte({
 			compilerOptions: {
-				hmr: true
+				hmr: false
```

Comment:
some HMR issues to work out still

Reply:
looks to be okay now

Reply:
We should probably leave HMR on during local playground testing by default?
```suggestion
				hmr: true
```